<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDB" >
      <attribute name="title" >
       <string>Database</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" colspan="2" >
        <widget class="QPushButton" name="btnListShow" >
         <property name="text" >
          <string>Get shows list</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QPushButton" name="btnCheckExpiredShows" >
         <property name="text" >
          <string>Check expired shows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <layout class="QFormLayout" name="formLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="lblId" >
           <property name="text" >
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="ldtShowId" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="lblName" >
           <property name="text" >
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="ldtShowName" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="lblUrl" >
           <property name="text" >
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="ldtUrl" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="lblCountry" >
           <property name="text" >
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="ldtCountry" />
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="lblStarted" >
           <property name="text" >
            <string>Started</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QLineEdit" name="ldtStarted" />
         </item>
         <item row="5" column="0" >
          <widget class="QLabel" name="lblEnded" >
           <property name="text" >
            <string>Ended</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <widget class="QLineEdit" name="ldtEnded" />
         </item>
         <item row="6" column="0" >
          <widget class="QLabel" name="lblEndedFlag" >
           <property name="text" >
            <string>Ended Flag</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" >
          <widget class="QCheckBox" name="cbxEndedFlag" >
           <property name="layoutDirection" >
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2" >
        <widget class="QPushButton" name="btnSaveShow" >
         <property name="text" >
          <string>Save this Show</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2" >
        <widget class="Line" name="line_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>debug information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <widget class="QTextEdit" name="infoTextEdit" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0" colspan="2" >
        <widget class="Line" name="line_4" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1" >
        <widget class="QLineEdit" name="ldtUpdateShowId" />
       </item>
       <item row="7" column="0" >
        <widget class="QPushButton" name="btnUpdateShow" >
         <property name="text" >
          <string>Update this show</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabView" >
      <attribute name="title" >
       <string>View</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="btnRequestDB" >
         <property name="text" >
          <string>Request DB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QTableView" name="tblView" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
